geocoder/google.py

```diff
@@ -182,7 +188,10 @@ def _build_params(self, location, provider_key, **kwargs):
             params['client'] = self.client
             return self._encode_params(params)
         # or API key
+        # In this case: raise exception if not valid key found
         else:
+            if not provider_key:
+                raise ValueError('Provide API Key')
```

Comment:
Wouldn't this raise an Error if you **don't** provide an API key?

Using the Google Provider an API key is optional